Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/openSAML/BuildEvidenceParameter.java

```diff
@@ -0,0 +1,111 @@
+package gov.hhs.fha.nhinc.callback.openSAML;
+
+import java.util.List;
+import org.joda.time.DateTime;
+import org.opensaml.saml.saml2.core.AttributeStatement;
+import org.opensaml.saml.saml2.core.Subject;
+
+public class BuildEvidenceParameter {
+
+    private CallbackProperties properties;
+    private String evAssertionID;
+    private DateTime issueInstant = null;
+    private String format = null;
+    private DateTime beginValidTime = null;
+    private DateTime endValidTime = null;
+    private String issuer = null;
+    private List<AttributeStatement> statements;
+    private Subject subject;
+
+    public BuildEvidenceParameter(String evAssertionID, String issuer, CallbackProperties properties,
+        List<AttributeStatement> statements, Subject subject) {
+        this.evAssertionID = evAssertionID;
+        this.properties = properties;
+        this.statements = statements;
+        this.subject = subject;
+        this.issuer = issuer;
+        setProperties(properties);
+    }
+
+    public CallbackProperties getProperties() {
+        return properties;
+    }
+
+    public void setProperties(CallbackProperties properties) {
+        setEvAssertionID(evAssertionID);
+        issueInstant = properties.getEvidenceInstant();
+        setIssueInstant(issueInstant);
+        format = properties.getEvidenceIssuerFormat();
+        setFormat(format);
```

Comment:
Question -- am I missing something here, or is everything being assigned twice? (once via direct assignment, once via the setter)